src/controllers/main.ts

```diff
@@ -186,3 +193,66 @@ export const getHelpController = async (
     next(error);
   }
 };
+
+/**
+ * list the pages handled with the external CMS
+ *
+ * Record of <internalSlug, externalSlug>
+ */
+const EXISTING_EXTERNAL_PAGES: Record<string, string> = {
+  accessibilite: "moncomptepro-accessibilite",
+  confidentialite: "moncomptepro-politique-de-confidentialite",
+  "conditions-generales": "moncomptepro-conditions-generales-d-utilisation",
+};
```

Comment:
don't really know where to put this: the list of URLS of the external CMS that we want to handle. Thoughts?

Reply:
🤷🏻‍♂️